mcs/class/referencesource/mscorlib/system/threading/threadpool.cs

```diff
@@ -644,7 +644,9 @@ internal void MarkThreadRequestSatisfied()
                 count = prev;
             }
         }
-
+#if MONO
+        internal static IntPtr THREADPOOL_WORKITEMS = InternalPerformanceCounter.GetImpl("Mono Threadpool", "Work Items Added");
```

Comment:
Width? Doesn't look good here..

Reply:
And now that I think about it, since we have DISABLE_PERFCOUNTERS on the native side this would crash when perfcounters are disabled.